tests/unittests/test_log_filtering_functions.py

```diff
@@ -61,4 +70,40 @@ def check_log_debug_with_user_logging(self, host_out: typing.List[str]):
         self.assertIn('logging info', host_out)
         self.assertIn('logging warning', host_out)
         self.assertIn('logging error', host_out)
+        # See HOST_JSON_TEMPLATE_WITH_LOGLEVEL_INFO, debug log is disabled
         self.assertNotIn('logging debug', host_out)
+
+    def test_info_with_sdk_logging(self):
+        """Invoke a HttpTrigger sdk_logging which contains logging invocation
+        via the azure.functions logger. This should be treated as system logs,
+        which means the log should not be displayed in local console.
+        """
+        r = self.webhost.request('GET', 'sdk_logging')
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.text, 'OK-sdk-logger')
+
+    def check_log_info_with_sdk_logging(self, host_out: typing.List[str]):
+        # See TestLogFilteringFunctions docstring
+        # System log should not be captured in console
+        self.assertNotIn('sdk_logger info', host_out)
+        self.assertNotIn('sdk_logger warning', host_out)
+        self.assertNotIn('sdk_logger error', host_out)
+        self.assertNotIn('sdk_logger debug', host_out)
+
+    def test_info_with_sdk_submodule_logging(self):
+        """Invoke a HttpTrigger sdk_submodule_logging which contains logging
+        invocation via the azure.functions logger. This should be treated as
+        system logs.
+        """
+        r = self.webhost.request('GET', 'sdk_submodule_logging')
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.text, 'OK-sdk-submodule-logging')
+
+    def check_log_info_with_sdk_submodule_logging(self,
+                                                  host_out: typing.List[str]):
+        # See TestLogFilteringFunctions docstring
+        # System log should not be captured in console
+        self.assertNotIn('sdk_submodule_logger info', host_out)
+        self.assertNotIn('sdk_submodule_logger warning', host_out)
+        self.assertNotIn('sdk_submodule_logger error', host_out)
+        self.assertNotIn('sdk_submodule_logger debug', host_out)
```

Comment:
Stretch: Any way to check that AppInsights logs as well - if it is being logged?

Reply:
Yeah, I actually tried to push traces to application insights, and fetch back the result from the GET https://api.applicationinsights.io/v1/apps/ API. The trace exists in Live Metrics tab, but they do not show up in the log analytics and cannot be queried from the REST API endpoint. So I think we will skip that.